bugs/sru-19.4.33-manual-4.txt

```diff
@@ -0,0 +1,306 @@
+=== Begin SRU Template ===
+[Impact]
+Images which provide multiple ssh authorized key files for users via
+setting AuthorizedKeysFiles = <file1> <file2> would result in an invalid ssh
```

Comment:
It's not an invalid SSH configuration; the fact that it's valid and we don't handle it is the problem we're fixing.